ouroboros-network-api/src/Ouroboros/Network/PeerSelection/LedgerPeers/Utils.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE BangPatterns     #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Ouroboros.Network.PeerSelection.LedgerPeers.Utils
+  ( bigLedgerPeerQuota
+  , accBigPoolStake
+  , reRelativeStake
+  , AccPoolStake (..)
+  , PoolStake (..)
+  , RelayAccessPoint (..)
+  ) where
+
+import Control.Exception (assert)
+import Data.Bifunctor (first)
+import Data.List (foldl', sortOn)
+import Data.List.NonEmpty (NonEmpty)
+import Data.Ord (Down (..))
+import Data.Ratio ((%))
+
+import Ouroboros.Network.PeerSelection.LedgerPeers.Type
+import Ouroboros.Network.PeerSelection.RelayAccessPoint
+
+-- | The total accumulated stake of big ledger peers.
+--
+bigLedgerPeerQuota :: AccPoolStake
+bigLedgerPeerQuota = 0.9
+
+-- | Sort ascendingly a given list of pools with stake,
+-- and tag each one with cumulative stake, with a cutoff
+-- at 'bigLedgerPeerQuota'
+--
+accBigPoolStake :: [(PoolStake, NonEmpty RelayAccessPoint)]
+                -> [(AccPoolStake, (PoolStake, NonEmpty RelayAccessPoint))]
+accBigPoolStake =
```

Comment:
Sine `accBigPoolStake` nand `reRelativeState` are now public API, it's better to give them better name, maybe:
* `accumulateBigLedgerState`
* `recomputeRelativeStake`